tests/draft2020-12/propertyNames.json

```diff
@@ -1,6 +1,7 @@
 [
     {
         "description": "propertyNames validation",
+        "specification": [ { "core": "10.3.2.4" } ],
```

Comment:
If we haven't finalized these additions (the discussion for which is elsewhere), we should hold off adding them here.  Please remove until that discussion had come to a final decision.

Reply:
Ok, cool with that 👍